tests/e2e/watched-namespaces/spark-example.yaml

```diff
@@ -27,6 +27,8 @@ spec:
     spark.executor.instances: "1"
     spark.kubernetes.container.image: "spark:4.0.0-preview1-scala2.13-java17-ubuntu"
     spark.kubernetes.authenticate.driver.serviceAccountName: "spark"
+    spark.kubernetes.driver.request.cores: "0.5"
+    spark.kubernetes.executor.request.cores: "0.5"
```

Comment:
This is to resolve the insufficient cpu issues when 2 operator pods running and a new spark application is placed